build/azDevOps/packages-amido-stacks-configuration.yml

```diff
@@ -20,7 +19,15 @@ variables:
     value: amidostacksuksprod
   - name: DevOps.RG
     value: amido-stacks-rg-uks-prod
-
+  # SelfConfig
+  - name: self_repo
+    value: stacks-dotnet-packages-configuration
+  - name: self_repo_dir
+    value: "$(Agent.BuildDirectory)/s/$(self_repo)"
+  - name: self_pipeline_repo
+    value: "$(Agent.BuildDirectory)/s/stacks-pipeline-templates"
+  - name: self_pipeline_scripts_dir
+    value: "$(self_pipeline_repo)/scripts"
```

Comment:
As this folder doesn't exist in the repo, how is it used by the yaml validation job?

Reply:
cancel this, i've just seen it is using the `stacks-pipeline-templates` repo